00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -75,14 +75,27 @@
    },
```

Comment:
<p>I would recommend here using <code>FRESNEL_MIN_VERSION</code>  instead of the leading underscore. In this case the variable functions more as a constant than a hidden or "private" variable. This is slightly more in line with typical Python variable casing. However, if you have strong feelings the other way that is fine, notebooks make variable naming in Python weirder than it already is since the notebook is designed to be interactive, and <em>hiding</em> variables is harder than it already was.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/24/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>